### ISSUE TITLE
[Bug] TypeError: ChatConfig.__init__() got an unexpected keyword argument 'estimated_vram_req'

### ISSUE THREAD

**Issue by arif599** [mlc-ai/mlc-llm#721]
## 🐛 Bug

I'm trying to run the rest api server using 
`python -m mlc_chat.rest --model mlc-chat-RedPajama-INCITE-Chat-3B-v1-q4f16_0`

```
(mlc-chat-venv) (env) LLM Server % python -m mlc_chat.rest --model mlc-chat-RedPajama-INCITE-Chat-3B-v1-q4f16_0
INFO:     Started server process [24490]
INFO:     Waiting for application startup.
[18:59:44] /Users/catalyst/Workspace/mlc-ai-package-self-runner/_work/package/package/tvm/src/runtime/metal/metal_device_api.mm:167: Intializing Metal device 0, name=Apple M1 Pro
System automatically detected device: metal
Using model folder: /Users/ar/Desktop/LLM Server/dist/prebuilt/mlc-chat-RedPajama-INCITE-Chat-3B-v1-q4f16_0
Using mlc chat config: /Users/ar/Desktop/LLM Server/dist/prebuilt/mlc-chat-RedPajama-INCITE-Chat-3B-v1-q4f16_0/mlc-chat-config.json
ERROR:    Traceback (most recent call last):
  File "/Users/ar/Desktop/LLM Server/env/lib/python3.11/site-packages/starlette/routing.py", line 677, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "/Users/ar/miniconda3/lib/python3.11/contextlib.py", line 204, in __aenter__
    return await anext(self.gen)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ar/Desktop/LLM Server/env/lib/python3.11/site-packages/mlc_chat/rest.py", line 102, in lifespan
    chat_mod = ChatModule(
               ^^^^^^^^^^^
  File "/Users/ar/Desktop/LLM Server/env/lib/python3.11/site-packages/mlc_chat/chat_module.py", line 566, in __init__
    self.chat_config = _get_chat_config(self.config_file_path, chat_config)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ar/Desktop/LLM Server/env/lib/python3.11/site-packages/mlc_chat/chat_module.py", line 285, in _get_chat_config
    final_chat_config = ChatConfig(**json_object)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: ChatConfig.__init__() got an unexpected keyword argument 'estimated_vram_req'

ERROR:    Application startup failed. Exiting.
(mlc-chat-venv) (env) LLM Server % 
```

## Environment

 - Platform (e.g. WebGPU/Vulkan/IOS/Android/**CUDA**):
 - Operating system (e.g. Ubuntu/Windows/**MacOS**/...):
 - Device (e.g. iPhone 12 Pro, PC+RTX 3090, ..., **macbook pro**)
 - How you installed MLC-LLM (`conda`, source, **pip**):
 - How you installed TVM-Unity (`pip`, **source**):
 - Python version (e.g. 3.10, **3.11**):
 - GPU driver version (if applicable):
 - CUDA/cuDNN version (if applicable):
 - TVM Unity Hash Tag (`python -c "import tvm; print('\n'.join(f'{k}: {v}' for k, v in tvm.support.libinfo().items()))"`, applicable if you compile models):
 - Any other relevant information:

**Comment by tqchen** [comment]
Seems we need to enable chatconfig to allow extra parameters that user might append @sudeepag @Kathryn-cat @CharlieFRuan

**Comment by MasterJH5574** [comment]
Fixed in #722. @arif599 would you like to try again? If the issue still exists you are welcome to create a new issue.
